pkgs/by-name/we/wemeet/package.nix

```diff
@@ -0,0 +1,130 @@
+{
+  pkgs,
```

Comment:
This parameter is unused.

Reply:
This should have been caught by the nixf-tidy CI check.  Hmm, why does that check not run for this PR?

Any ideas? @Aleksanaa

Reply:
Maybe the PR needs to be rebased

Reply:
fixed and rebased